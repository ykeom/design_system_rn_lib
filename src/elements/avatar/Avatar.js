import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Avatar extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>avatar</Text>
      </View>
    );
  }
}

export default Avatar;
