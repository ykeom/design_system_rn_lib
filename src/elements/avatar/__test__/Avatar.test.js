import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Avatar from '../Avatar';

describe('Avatar', () => {
  it('should render', () => {
    const component = shallow(<Avatar />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
