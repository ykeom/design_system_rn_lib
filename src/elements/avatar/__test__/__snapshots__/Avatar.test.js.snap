// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    avatar
  </Text>
</View>
`;
